.travis.yml

```diff
@@ -1,4 +1,3 @@
-sudo: false
 language: ruby
 script: bundle exec script/test
 cache: bundler
```

Comment:
I noticed all failing tests have some issues with bundler.
Maybe removing the cache for it will fix them up?

Reply:
It’s worth a try!